_challenges/coding-in-the-cabana/index.md

```diff
@@ -8,4 +8,4 @@ redirect_from:
   - /CodingInTheCabana/index.html
 ---
 
-The sun out is, the birds are chirping, it's a beautiful day to code a generative algorithm. Choo choo!
+The sun is out, The birds are chirping and it's a beautiful day to code a generative algorithm. Choo Choo!
```

Comment:
Thanks for catching the typeo in the first phrase but I'm not sure about the other changes. Could you explain them?